cypress/run-e2e.sh

```diff
@@ -17,16 +17,12 @@ finally() {
   local hanging_server_processes=$(ps aux | grep -v grep | grep 'node production-start.js' | awk '{print $2}')
 
   # Send kill signal to the processes
-  if [ ${#hanging_api_processes} -gt "0" ];  then
-    kill -9 $hanging_api_processes &>/dev/null
+ local process= [ ${#hanging_client_processes} -gt "0" ] && process=$hanging_client_processes || process=$hanging_api_processes  
```

Comment:
Doesn't this logic mean that process is either the client or the api processes, but never both? I think it's likely that both will still be present.

Also, this currently errors out with

```
End to end bash script exiting gracefully
./cypress/run-e2e.sh: line 20: local: `[': not a valid identifier
./cypress/run-e2e.sh: line 20: local: `34': not a valid identifier
./cypress/run-e2e.sh: line 20: local: `-gt': not a valid identifier
./cypress/run-e2e.sh: line 20: local: `0': not a valid identifier
./cypress/run-e2e.sh: line 20: local: `]': not a valid identifier
```

I don't really know enough bash to say what's going on, sorry!

Reply:
It essentially tests if the first process isn't empty, in the case that it isn't empty it successfully stores it and continues the program skipping the "or" statement that follows. However, in the case, that it is empty the local process will be set to the next one that follows. The following if condition then checks if everything before is empty and gets set to the client process. This is under the finally() function, as long as it is called it will execute whether the promise is successful or rejected.